.github/workflows/docker-build.yml

```
@@ -0,0 +1,99 @@
+        make docker-build
+        # Tag for consistent naming
+        docker tag document-query:latest document-query:${{ inputs.image-tag }}
+
+    - name: Test Docker image
+      run: |
+        # Start container in background
+        docker run --rm -d --name test-app -p 8080:8000 \
+          -e SECRET_KEY=test-secret-key \
+          -e DEBUG=0 \
+          document-query:${{ inputs.image-tag }}
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)